spec/models/miq_policy/import_export_spec.rb

```diff
@@ -0,0 +1,31 @@
+describe MiqPolicy::ImportExport do
+  # let(:policy) { (Class.new { include MiqPolicy::ImportExport }).new }
+
+  context '.import_from_hash' do
+
+    it "loads attributes" do
+      p_hash = {
+        "name"             => "t_name",
+        "description"      => "t_description",
+        "expression"       => nil,
+        "towhat"           => "ContainerImage",
+        "guid"             => "e7a270bc-109b-11e6-86ba-02424d459b45",
+        "mode"             => "control",
+        "read_only"        => true,
+        "MiqPolicyContent" => []
+      }
+      policy, _ = MiqPolicy.import_from_hash(p_hash)
+      expect(policy).to have_attributes(p_hash.except("MiqPolicyContent"))
```

Comment:
Can you add the check for the nil "active" becoming `true` here?

`expect(policy.active).to be_truthy`